app/config/featureSwitches/FeatureSwitching.scala

```diff
@@ -50,6 +51,25 @@ trait FeatureSwitching {
     })(LocalDateTime.parse(_))
   }
 
+  def setEstimatedLPP1FilterEndDate(dateTimeToSet: Option[LocalDateTime]): Unit = {
+    logger.debug(s"[FeatureSwitching][setEstimatedLPP1FilterEndDate] - setting estimated LPP1 filter end date to: $dateTimeToSet")
+    dateTimeToSet.fold(sys.props -= ESTIMATED_LPP1_FILTER_END_DATE)(sys.props += ESTIMATED_LPP1_FILTER_END_DATE -> _.toString)
+  }
+
+  def getEstimatedLPP1FilterEndDate: LocalDateTime = {
+    sys.props.get(ESTIMATED_LPP1_FILTER_END_DATE).fold({
+      val optDateAsString = config.getOptional[String]("feature.switch.estimated-lpp1-filter-end-date")
+      val dateAsString = optDateAsString.getOrElse("")
+      if (dateAsString.isEmpty) {
+        LocalDateTime.now().minusDays(1)
```

Comment:
Today minus one day doesn't sound right to me... maybe its worth changing withinLPP1FilterWindow so that it checks 'now is on or before the end date'?

Reply:
Changed